public/assets/styles/site.css

```diff
@@ -0,0 +1,195 @@
+/* Oh Well */
+
+:root {
+  --foreground: #FFFFFF;
+  --background: #212121;
+  --background-dark: #000000;
+  --dark-accent-0: #444444;
+  --dark-accent-1: #333333;
+  --dark-accent-2: #888888;
+  --dark-accent-3: #4E4E4E;
+  --background-dark-transparent: #0000004d;
+  --red: #FF0000;
+}
+
+
+/* Define a rule for our custom font */
+@font-face {
+  font-family: 'badhandwriting';
+  font-style: normal;
+  font-weight: normal;
+  src: local('badhandwriting'),
+    url('fonts/badhandwriting.woff') format('woff'),
+    url('fonst/badhandwriting.ttf') format('truetype');
+}
```

Comment:
This `@font-face` designation uses both woff and truetype; 
A bunch of browsers are against using truetype; so it's always smart to have a backup; woff is pretty universal, though not as crisp as TTF or SVG fonts.